src/components/service.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
```

Comment:
You're mostly using pixel units here. It's more appropriate to use rem units

Reply:
You should also run a formatter on this file, some inconsistencies
e.g
```
div{
...

.textBackground {
```

Reply:
Formatting tools will be set up soon, so this isn't an issue at this point.